### PR TITLE
events: add foreman-probing deep dive

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

File: _data/events.yml [hunk 1]
```diff
@@ -1236,3 +1236,18 @@
     check the site. Presenter Lukas Zapletal from the Foreman Core team
     can speak Czech or English. These are the languages of choice for the
     event. No hardware should be required, lab with 24 PCs.
+- title: 'Foreman Deep Dives: Foreman-Probing (with Adam Ruzika)'
+  type: 'Livestream'
+  location: virtual
+  start: 2018-03-07 14:00 GMT
+  end: 2018-03-07 15:00 BST
```
Comment (dLobatog) [inline _data/events.yml]: Is this supposed to be really BST?

**Review by dLobatog — CHANGES_REQUESTED** [review]
Thanks @orrabin ! Check out the comment, I'm not sure about that?

File: _data/events.yml (hunk 1 above)
Comment (orrabin) [reply _data/events.yml]: nope, copy+past problem, I'll fix it, thanks!

**Comment by orrabin** [comment]
@dLobatog I fixed the timezone, thanks

**Comment by mmoll** [comment]
@orrabin this needs a rebase now. Also, this branch was created in the main repo rather in your fork. ;)

**Comment by orrabin** [comment]
@mmoll that's because I did it directly from github but I've opened: https://github.com/theforeman/theforeman.org/pull/1028 instead.

**Pull request closed without merging**